Script integracion a DB

oscarCR19 opened the pull request.

oscarCR19 committed:
Script integracion a DB

tecnologer commented on ToDoListScrip.sql:
```diff
@@ -0,0 +1,14 @@
+create database ToDoList
```
falta el `;`

tecnologer commented on ToDoListScrip.sql:
```diff
@@ -0,0 +1,14 @@
+create database ToDoList
+create table ToDoList.Usuarios(ID int not null, 
```
hay que usar identacion para darle un formato legible

tecnologer commented on ToDoListScrip.sql:
```diff
@@ -0,0 +1,14 @@
+create table ToDoList.Usuarios(ID int not null, 
+Nombre citext not null unique, 
+PRIMARY KEY(ID))
+
```
no hay definicion de usuario(s)

oscarCR19 replied:
Hola buenas tengo duda a que se refiere con definición de usuarios, por el momento dicha table consta de un ID  y un nombre, con estos atributos se pueden identificar los usuarios.

tecnologer replied:
cuando defines una base de datos mediante script, debes de crear la configuracion de los usuario sque tendran acceso a dicha db, normalmente se crean dos usuarios:

- un superadmin (sa) 
- un develop

el sa tiene full access, mientras que el develop solo puede insertar, actualizar y leer los datos, no puede modificar tablas o crear otros usuarios.

oscarCR19 committed:
Add files via upload

Este es el script actualizado.
Se implemento la creación de usuarios y sus privilegios correspondientes.
Se mejoró la indentación del script.

tecnologer commented on ToDoListScrip.sql:
```diff
@@ -1,14 +1,19 @@
-FechaCompletado timestamp with timezone,
-PRIMARY KEY(ID),
-FOREIGN KEY(UsuarioID) reference Usuarios(ID) );
+create table Usuarios(ID int not null, 
```
y el create database?

oscarCR19 committed:
Add files via upload

Buenos dias
Disculpe el inconveniente, no me percaté del error, ya el .sql esta listo

tecnologer reviewed (approved):
hay que mejorar un poco con la identacion, pero es un muy buen comienzo
